egs/csj/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+
```

Comment:
can you please ensure there are some results in the tops of these files, so people don't have to check the RESULTS files?  You can make a script to display and compare them if you want. 
Make sure that any results that do appear in the RESULTS file are up to date.

Reply:
Sure, the experiment is running. I will update later. 
And I prefer to provide a compare_wer_general.sh to do that.
The RESULTS file maybe need to updata, because it comes from non-speed-perturbed feature.